### DISCUSSION TITLE
The repository ismaildedec has been retired and cannot be reused.

### DISCUSSION THREAD

**Discussion by ismaildedec** [community/community#148034] [New to GitHub]
I am requesting to create github repositories. The repository ismaildedec has been retired and cannot be reused. gives me this error. How will I fix this?

**Comment by davevad93** [comment]
Hi @ismaildedec ,

In order to fix this, you have to contact support and ask them to unretire that repository name for you, open a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue.
